### PR TITLE
14.0 mig website crm privacy policy

### PR DESCRIPTION
**Pull request by pedroguirao**

### CODE REVIEW

File: website_crm_privacy_policy/templates/contactus.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<odoo>
+    <!-- Privacy Policy -->
+    <template
+        id="contactus_form"
+        inherit_id="website_crm.contactus_form"
+        name="Add privacy policy checkbox"
+    >
+        <xpath
+            expr="//form//div/div/span[@id='s_website_form_result']/../.."
```
Comment (victoralmau) [inline website_crm_privacy_policy/templates/contactus.xml]:
```suggestion
            expr="//form//div/div/span[@id='s_website_form_result']/.."
```

File: website_crm_privacy_policy/templates/contactus.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            position="before"
+        >
+            <div class="form-group row form-field s_website_form_required_custom">
+                <div class="offset-lg-3 offset-md-4 col-md-8 col-lg-7">
```
Comment (victoralmau) [inline website_crm_privacy_policy/templates/contactus.xml]:
```suggestion
                <div class="col-sm" style="margin-left: 200px;">
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
It is important to put this block in the correct place.
Now:
antes

With my suggestions:
despues

File: website_crm_privacy_policy/templates/contactus.xml (hunk 2 above)
Comment (pedroguirao) [reply website_crm_privacy_policy/templates/contactus.xml]: Thank you Victor! its makes sense

**Comment by victoralmau** [comment]
Please, can you unify your commits into one (Migration to 14.0)?

**Comment by pedroguirao** [comment]
Hi Victor, first timer here, have to close this push and do a new one? or there is another way? Thank you

**Comment by victoralmau** [comment]
In this same PR you must fix up those commits of yours in just one.

**Comment by pedrobaeza** [comment]
https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by pedroguirao** [comment]
Thank you Victoralmau and Pedrobaeza, hope it's OK

**Comment by JordiBForgeFlow** [comment]
@pedroguirao can you close this PR?

**Comment by pedroguirao** [comment]
Sorry @JordiBForgeFlow, Im going to close it now.

**Pull request closed without merging**